server/middleware/barcode/setupBarcode.ts

```diff
@@ -89,6 +89,7 @@ export default function setUpCreateBarcode(
   router.get('/generate-barcode-image', (req, res) => generateImageController.getGenerateImageView(req, res))
 
   router.get('/edit-contact/:contactId', (req, res) => editContactController.getEditContact(req, res))
+  router.post('/edit-contact', (req, res) => editContactController.submitUpdateContact(req, res))
```

Comment:
Did you consider making this POST endpoint `/edit-contact/:contactId` like the GET endpoint? A bit more restful (though of course its a web app, so is not rest at all 😁  )